test/seq/runtests.jl

```diff
@@ -3200,4 +3217,15 @@ end
     end
 end
 
+@testset "majorityvote" begin
+    dnas = [dna"CTCGATCGATCC", dna"CTCGAAAAATCA", dna"ATCGAAAAATCG", dna"ATCGGGGGATCG"]
+    dna2 = [dna"CTCGATCGATCC", dna"CTCGAAA", dna"ATCGAAAAATCG", dna"ATCGGGGGATCG"]
+    rnas = [rna"CUCGAUCGAUCC", rna"CUCGAAAAAUCA", rna"AUCGAAAAAUCG", rna"AUCGGGGGAUCG"]
+    rna2 = [rna"CUCGAUCGAUCC", rna"CUCGAAAAAUCA", rna"AUCGAAAAAUCG", rna"AUCGUCG"]
+    @test majorityvote(dnas) == dna"MTCGAAARATCG"
+    @test majorityvote(rnas) == rna"MUCGAAARAUCG"
+    @test_throws ArgumentError majorityvote(dna2)
+    @test_throws ArgumentError majorityvote(rna2)
+end
```

Comment:
I think you need to test a few pathological cases (e.g. `DNASequence[]`, `[dna"", dna""]`)